src/storage/kv/store.rs

```diff
@@ -700,11 +700,16 @@ impl Core {
 
             let index_value = encode_entry(entry);
 
+            let ts = if entry.ts == 0 {
```

Comment:
I was wondering if we need these conditional checks at all given that we are setting the commit ts for each entry

```
    fn assign_commit_ts(&mut self) -> u64 {
        let commit_ts = now();
        self.write_set.iter_mut().for_each(|entry| {
            if entry.ts == 0 {
                entry.ts = commit_ts;
            }
```

I think the commit_ts from task should be removed completely, just so simplify this so that we can just pass entry.ts everywhere. My bad I did not figure it out before. Do you think it is worth doing in this PR? Because else you have to add these checks everywhere